cmd/local-artifact-mirror/pull.go

```diff
@@ -129,15 +129,40 @@ var helmChartsCommand = &cli.Command{
 		dst := defaults.EmbeddedClusterChartsSubDir()
 		src := filepath.Join(location, HelmChartsArtifactName)
 		logrus.Infof("uncompressing %s", src)
-		if err := tgzutils.Decompress(src, dst); err != nil {
+		newFiles, err := tgzutils.Decompress(src, dst)
+		if err != nil {
 			return fmt.Errorf("unable to uncompress helm charts: %w", err)
 		}
 
+		// check which files are present in the directory but not newFiles, and remove them
+		files, err := os.ReadDir(dst)
+		if err != nil {
+			return fmt.Errorf("unable to read directory: %w", err)
+		}
+		for _, f := range files {
+			if !contains(newFiles, f.Name()) {
```

Comment:
can we use https://pkg.go.dev/slices#Contains instead ?

Reply:
Yes